src/gui/base/Checkbox.ts

```diff
@@ -37,10 +37,14 @@ export class Checkbox implements Component<CheckboxAttrs> {
 			},
 			[
 				m(
-					".wrapper.flex.items-center",
+					`.wrapper.flex.items-center${a.disabled ? ".disabled" : undefined}`,
```

Comment:
Replace undefined by an empty string, otherwise it will append ```undefined``` when **a.disabled** is false

Reply:
Nice catch, pushed a commit now.